### PR TITLE
Custom Rake tasks

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

File: content/posts/custom-rake-tasks.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  end
+end
+```
+
+## `.rake` files
+
+If your list of Rake tasks is growing, you may want to split them per categories in several files:
+
+```shell
+$ mkdir -p rakelib
```
Comment (AlfonsoUceda) [inline content/posts/custom-rake-tasks.md:36]: one thing, @jodosha is this folder automatically loaded? I mean when you call `rake -T`

Comment (jodosha) [reply content/posts/custom-rake-tasks.md:36]: Yes, I just verified manually with a testing Hanami project.

File: content/posts/custom-rake-tasks.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+```ruby
+# rakelib/catalog.rake
+namespace :catalog do
```
Comment (davydovanton) [inline content/posts/custom-rake-tasks.md:41]: also, I found that you need to load `config/boot.rb` in some cases (when you work with dry-system for example. WDYT to adding this point too?

**Review by davydovanton — APPROVED** [review]

**Review by AlfonsoUceda — APPROVED** [review]

File: content/posts/custom-rake-tasks.md (hunk 1 above)
Comment (davydovanton) [reply content/posts/custom-rake-tasks.md:36]: yep, I use custom rake task in my project and this folder works without any issues

File: content/posts/custom-rake-tasks.md (hunk 2 above)
Comment (jodosha) [reply content/posts/custom-rake-tasks.md:41]: Do you have an example?

Comment (davydovanton) [reply content/posts/custom-rake-tasks.md:41]: will make it in few seconds

Comment (davydovanton) [reply content/posts/custom-rake-tasks.md:41]: ## without booting
screenshot 2018-11-20 18 06 35
screenshot 2018-11-20 18 06 39

## with booting
screenshot 2018-11-20 18 06 58
screenshot 2018-11-20 18 07 02

Comment (jodosha) [reply content/posts/custom-rake-tasks.md:41]: @davydovanton Adding `:environment` as dependency of the task doesn't do the trick?

```ruby
task tasks: :environment do
  # ...
end
```

Can you please tell me if it works? Thanks!

File: content/posts/custom-rake-tasks.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+preview: "How to add custom Rake tasks"
+status: "Learn how to add custom Rake tasks to your Hanami project"
+date: 2018-11-20T14:49:44+01:00
+draft: false
+gems: ["hanami"]
+tags: ["hanami", "rake"]
+versions: ["1.0"]
+---
+
+If you want to add customer Rake tasks to your Hanami project, you have two ways: `Rakefile` and `.rake` files.
```
Comment (kaikuchn) [inline content/posts/custom-rake-tasks.md]: Minor typo here:

```suggestion
If you want to add custom Rake tasks to your Hanami project, you have two ways: `Rakefile` and `.rake` files.
```

File: content/posts/custom-rake-tasks.md (hunk 2 above)
Comment (davydovanton) [reply content/posts/custom-rake-tasks.md:41]: checked, all works correct 👍

Comment (jodosha) [reply content/posts/custom-rake-tasks.md:41]: Thanks!

**Pull request merged**